### PR TITLE
Add clients API coverage page

### PR DESCRIPTION
**Pull request by ionut-arm**
This commit adds a new page to the book describing the API coverage of
the client libraries.

Signed-off-by: Ionut Mihalcea <ionut.mihalcea@arm.com>

### CODE REVIEW

File: src/clients_api_coverage.md [hunk 1]
```diff
@@ -0,0 +1,71 @@
...
+the [PSA Cryptography API version
+1.0](https://developer.arm.com/architectures/security-architectures/platform-security-architecture/documentation).
+This page describes the current state of support within clients for the operations, attributes and
+options that are defined in that specification and for Parsec-related functionality (e.g. support
+for various providers).
+
+For an overview of the functionality supported by the service, see the [service API coverage
+page](service_api_coverage.md).
+
+**Note:** Only client libraries that are under our control are represented in the tables below. This
```
Comment (hug-dev) [inline src/clients_api_coverage.md:13]: Maybe "under our control" -> "under the Parsec organization control" or something similar?

File: src/clients_api_coverage.md [hunk 2]
```diff
@@ -0,0 +1,71 @@
...
+| SHA-256     | ✅         |
+| SHA-384     | ✅         |
+| SHA-512     | ✅         |
+| SHA-512-224 | ✅         |
+| SHA-512-256 | ✅         |
+| SHA3-224    | ✅         |
+| SHA3-256    | ✅         |
+| SHA3-384    | ✅         |
+| SHA3-512    | ✅         |
+| Any hash    | ✅         |
```
Comment (hug-dev) [inline src/clients_api_coverage.md:58]: "Any hash" can be removed now I think as it has been removed from the operations

File: src/clients_api_coverage.md (hunk 1 above)
Comment (ionut-arm) [reply src/clients_api_coverage.md:13]: Is it Parsec organization or ParallaxSecond organisation? Or something else? 😕

Comment (hug-dev) [reply src/clients_api_coverage.md]: I'd say Parsec is it is the title of the `parallaxsecond` organization page 😬

**Pull request merged**